app/src/main/java/de/danoeh/antennapod/adapter/DownloadStatisticsListAdapter.java

```diff
@@ -0,0 +1,125 @@
+package de.danoeh.antennapod.adapter;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.RequestOptions;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.core.glide.ApGlideSettings;
+import de.danoeh.antennapod.core.storage.DBReader;
+import de.danoeh.antennapod.core.util.Converter;
+import de.danoeh.antennapod.view.PieChartView;
+
+/**
+ * Adapter for the statistics list
+ */
+public class DownloadStatisticsListAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
```

Comment:
This class is basically the same as the other adapter. Could you please use a common parent class to reduce code duplication? This makes it easier to update the class and add stuff like adding colorful dots, as discussed here: https://github.com/AntennaPod/AntennaPod/pull/3485#issuecomment-541055521

<img width="150" src="https://user-images.githubusercontent.com/5811634/66653327-0c49d600-ec38-11e9-9052-cb13353a4563.png" />